providers/AVISO.json

```diff
@@ -123,6 +123,83 @@
         }
     },
     "elevation": {
+        "FES2012": {
+            "format": "FES-netcdf",
+            "constituents": [
+                "t2",
+                "msf",
+                "mtm",
+                "l2",
+                "mm",  
+                "q1",
+                "2n2",
+                "n4",
+                "o1",
+                "m3",
+                "n2",
+                "k1",
+                "s1",
+                "ms4",
+                "r2",
+                "mu2",
+                "eps2",
+                "s4",
+                "ssa",
+                "l2",
+                "mn4",
+                "m2",
+                "j1",
+                "m6",                
+                "mf",
+                "m4",
+                "p1",  
+                "k2",
+                "mks2",
+                "s2",
+                "m8",   
+                "nu2"
+            ],
+            "model_file": [
+                "fes2012/data/T2_FES2012_SLEV.nc",
+                "fes2012/data/MSf_FES2012_SLEV.nc",
+                "fes2012/data/Mtm_FES2012_SLEV.nc",
+                "fes2012/data/L2_FES2012_SLEV.nc",                          
+                "fes2012/data/Mm_FES2012_SLEV.nc",
+                "fes2012/data/Q1_FES2012_SLEV.nc",
+                "fes2012/data/2N2_FES2012_SLEV.nc",
+                "fes2012/data/N4_FES2012_SLEV.nc",
+                "fes2012/data/O1_FES2012_SLEV.nc",
+                "fes2012/data/M3_FES2012_SLEV.nc",
+                "fes2012/data/N2_FES2012_SLEV.nc",
+                "fes2012/data/K1_FES2012_SLEV.nc",
+                "fes2012/data/S1_FES2012_SLEV.nc",
+                "fes2012/data/MS4_FES2012_SLEV.nc",
+                "fes2012/data/R2_FES2012_SLEV.nc",
+                "fes2012/data/Mu2_FES2012_SLEV.nc",
+                "fes2012/data/E2_FES2012_SLEV.nc",
```

Comment:
Is this a match for `eps2`?

Reply:
I removed the constituents from the database and am running the files through the parser as before. Should make maintenance easier :)